src/components/views/rooms/ReplyTile.tsx

```diff
@@ -79,14 +79,27 @@ export default class ReplyTile extends React.PureComponent<IProps> {
 
     private onClick = (e: React.MouseEvent): void => {
         const clickTarget = e.target as HTMLElement;
-        // Following a link within a reply should not dispatch the `view_room` action
-        // so that the browser can direct the user to the correct location
-        // The exception being the link wrapping the reply
-        if (
+
+        // Clicking an image should open it and
+        // should not dispatch the `view_room` action
+        if (clickTarget.tagName.toLowerCase() === "img") {
+            e.preventDefault();
+        } else if (
+            clickTarget.classList.contains("mx_SeekBar") ||
+            clickTarget.getAttribute("data-test-id") === "play-pause-button"
```

Comment:
err, we shouldn't be relying on test IDs in production code. Can we not check for the voice message container?

Reply:
I change the test ID for a class test as I expect to be moved in the timeline when I click a non-interactive element of the voice message (e.g clock)